guides/v2.2/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -108,7 +108,7 @@ Any change not listed below is considered a PATCH level change.
 | | Changed event ordering | MAJOR|
 | **Virtual Type (marked with `@api`)**| Virtual type removed | MAJOR|
 | | Virtual type added | MINOR|
-| **URL Paths (Store-Front and Admin, any HTTP method)** | Path removed | MAJOR|
+| **URL Paths (Store-Front and Admin, GET)** | Path removed | MAJOR|
```

Comment:
Why is removing other types of requests not MAJOR as well?

Reply:
We didn't come to an agreement on this. I'll prepare a separate PR for POST requests to not block this one.